src/Node.py

```diff
@@ -154,10 +160,26 @@ def receive_message(self):
             message = self.retrieve_broadcast_message(priority_node)
 
             if message is not None:
-                message = message.parse_message_state(message)
+                message = message.parse_message_state(message) # message is an array of 2 arrays, the first being the voted values and the second the accepted values
                 self.process_received_message(message)
                 self.update_statement_count(priority_node, message)
                 log.node.info('Node %s retrieving messages from his highest priority neighbor Node %s!', self.name,priority_node.name)
+
+                voted_val = message[0] # message[0] is voted field
```

Comment:
why only the first one?

Reply:
The message varaible is structued like [[Value.hash 182...,etc.], [Value.hash: 273143]]. The first value [0] of message are all of the 'voted' values, and the second [1] are all of the 'accepted' values. That's why they are extracted in this way.